examples/birkhoff.jl

```diff
@@ -94,7 +96,7 @@ function build_birkhoff_lmo()
 end
 
 lmo = build_birkhoff_lmo()
-x, _, _ = Boscia.solve(f, grad!, lmo, verbose=true)
+x, _, _ = Boscia.solve(f, grad!, lmo, verbose=true, lazy=false, variant=Boscia.DICG())
```

Comment:
It would be nice to have both variants here. Also, it can function as a nice example for using `DICG`.